src/profile/Login.js

```diff
@@ -13,88 +13,88 @@ import logo from "../assets/Pareto_Lockup-01.png";
  * @TODO GH Issue #12
  */
 
-export default class Login extends Component {
-  constructor(props) {
-    super(props);
+const Login = ({
+  initialFetch,
+  setCloseLoading,
+  setLoading,
+  userHasAuthenticated,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
 
-    this.state = {
-      isLoading: false,
-      email: "",
-      password: "",
-    };
-  }
+  const [values, setValues] = useState({
+    email: "",
+    password: "",
+  });
 
-  validateForm() {
-    return this.state.email.length > 0 && this.state.password.length > 0;
-  }
+  const validateForm = () => {
+    return values.email.length > 0 && values.password.length > 0;
+  };
 
-  handleChange = (event) => {
-    this.setState({
-      [event.target.id]: event.target.value,
-    });
+  const handleChange = (event) => {
+    setValues({ ...values, [event.target.id]: [event.target.value] });
```

Comment:
This is mostly right, but you do not need a bracket around the event.target.value. If you go to the handleSubmit, and you console.log the values - you will find that the string is inside of an array, which triggers an Error from Amplify's text validation. 

Please remove the brackets in another commit, and push it up. After that, I will accept the PR - it's working outside of that. Then, go ahead and make an account for yourself after pulling the new changes from the upstream `main` branch. I have re-enabled self-service registration!

Reply:
Thanks for catching that! I've gone ahead and made that change.